common/export.py

```diff
@@ -389,6 +389,22 @@ def format_json(json_path):
     return subprocess.run(f"npm run fix:prettier {json_path}".split(" "))
 
 
+def display_changes_from_json(
```

Comment:
Not sure to understand the meaning of this function, when taken alone as is.
By stacking helper functions intended for reuse, we tend to loose the upper flow of the main function.

We should force us to write pure functions, which are easier to reason about and easier to test.
And keep the effectful treatments close to the top calling function.

Reply:
This function has just been extracted from the `export_processes_to_dirs` to achieve the goal you’re mentioning. Having some code that was displaying changes in a function named `export_processes_to_dirs` that should only write json looked strange to me (and it allows to run it only for one json, and not for all the exported json).